www/js/app.js

```diff
@@ -184,6 +184,7 @@ document.addEventListener('DOMContentLoaded', function () {
     getDefaultLanguageAndTranslateApp();
     resizeIFrame();
     abstractFilesystemAccess.loadPreviousZimFile();
+    handlePrintButton();
```

Comment:
Don't add this here. See below for explanation. You don't need to wait for DOMContentLoaded to attach events for static elements like the print button.